app/models/user.rb

```diff
@@ -47,7 +47,8 @@
 #  name                        :string(255)
 #  ninja_banned                :boolean          default(FALSE)
 #  password_digest             :string(255)      default(""), not null
-#  past_names                  :string           default([]), not null, is an Array
+#  past_names                  :string           default([]), not null,
+#                                                          is an Array
```

Comment:
Try to avoid manual changes to this section, it's maintained by the `annotate` gem